### PR TITLE
Fix Margin in Session Pages

### LINKED ISSUE #356: [Assigned] Bottom margin in My Sessions page is too small
## Overview (Required)

20180120_185511_screenshot_2018-01-20-18-54-39 jpg 1080x1920

### PR DESCRIPTION
**Pull request by yt-tkhs**
## Issue
- close #356 

## Overview (Required)
- Fix margin in AllSessions, RoomSessions, and FavoriteSessions.
- Add bottomMargin 4dp and topMargin 10dp to RecyclerView.
- Modify Divider to 12dp.
- Remove margin at Date Header.

## Links
- None

## Screenshot
Before | After
:--: | :--:
 | 
 | 
 |

### CODE REVIEW

**Comment by yt-tkhs** [comment]
I just removed WIP tag, so please review this PR 🙇

**Comment by takahirom** [comment]
Since it is not detailed about circumstances, please review @konifar  🙏

File: app/src/main/res/layout/fragment_all_sessions.xml [hunk 1]
```diff
@@ -30,7 +30,10 @@
             style="@style/BaseRecyclerView"
             android:layout_width="0dp"
             android:layout_height="0dp"
+            android:clipToPadding="false"
```
Comment (konifar) [inline app/src/main/res/layout/fragment_all_sessions.xml]: 👍

**Comment by konifar** [comment]
Looks good 👍  Thanks!

**Pull request merged**